scripts/jenkins/ardana/ansible/roles/input_model_generator/templates/input_model/data/networks.yml

```diff
@@ -36,10 +36,20 @@
 {%   if network_group['tagged_vlan']|default(True) %}
       vlanid: {{ ns.net_id }}
 {%   endif %}
-      tagged-vlan: {{ network_group['tagged_vlan']|default('True')|lower }}
+      tagged-vlan: {{ network_group['tagged_vlan']|default('False')|lower }}
+{%   if ipv6 == true %}
+{%   if ns.net_id != 100 %}
+      cidr: fc00:0:0:{{ ns.net_id }}::/64
```

Comment:
If not a big bother, we can make this to be fd00:0.0... fc addresses were reserved for use by allocation authority :P . https://en.wikipedia.org/wiki/Unique_local_address